test/metabase/metabot/command_test.clj

```diff
@@ -40,7 +40,7 @@
 ;; ok, now let's look at the actual response in its entirety
 (tt/expect-with-temp [Card [{:keys [name id]}]
                       Card [_ {:archived true}]]
-  {:response (format "Here's your 1 most recent cards:\n%d.  <https://metabase.mysite.com/question/%d|\"%s\">"
+  {:response (format "Here are your 1 most recent cards:\n%d.  <https://metabase.mysite.com/question/%d|\"%s\">"
```

Comment:
can this test be converted to a `deftest` since you're already touching it, and can you add a test for the `zero? card-count` condition you added?

Reply:
example to replace this (maintaining the with-temp) https://github.com/metabase/metabase/blob/master/test/metabase/pulse_test.clj#L736-L752

you want to use the `with-temp*` form since you have multiple definitions